### DISCUSSION TITLE
having problem with debugger

### DISCUSSION THREAD

**Discussion by amirkian** [serenity-is/Serenity#5319] [Help]
it was used to work correctly before but These days when I put **_debugger_** in my ts file ,an error will occurs that says

_Could not load content for http://localhost:8080/Modules/LifePlus/ApplicantAddressHistory/ApplicantAddressHistoryEditDialog.ts (HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE)_

![debugger](https://user-images.githubusercontent.com/19525693/102986292-d683cb00-4525-11eb-85a4-d92787941803.jpg)

**Comment by VictorTomaili** [comment]
I transfer this issue to discussion, if any bug found on this discussion, feel free convert back to issue.

**Reply by VictorTomaili** [reply]
and please provide some of info about your serenity version, typescript configuration, typescript version, browser version, visual studio version etc.
